### ISSUE TITLE
 Uncaught TypeError: window.queueMicrotask

### ISSUE THREAD

**Issue by Pirol62** [thomasloven/hass-browser_mod#168]
My Home Assistant version: 0.118.3

What I am doing:
Nothing special - normal usage

What I expected to happen:

What happened instead:

**Minimal** steps to reproduce:

```yaml
# The least ammount of code possible to reproduce my error



# End of code
```

Error messages from the browser console:

````
Logger: frontend.js.latest.202010214
Source: components/system_log/__init__.py:193
First occurred: 13:33:38 (183 occurrences)
Last logged: 14:03:48

https://nauen.duckdns.org:8123/browser_mod.js:149:7364 Uncaught TypeError: window.queueMicrotask is not a function
Home Assistant wurde vollständig gestartet!

````
**By putting an X in the boxes ([ ]) below, I indicate that I:**

- [x ] Understand that this is a channel for reporting bugs, not a support forum (https://community.home-assistant.io/).

- [x ] Have made sure I am using the latest version of the plugin.

- [x ] Have followed the troubleshooting steps of the "Common Problems" section of https://github.com/thomasloven/hass-config/wiki/Lovelace-Plugins.

- [ ] Understand that leaving one or more boxes unticked or failure to follow the template above may increase the time required to handle my bug-report, or cause it to be closed without further action.

**Comment by Pirol62** [comment]
I received two errors during restart:
````
Logger: homeassistant.components.websocket_api.http.connection
Source: components/websocket_api/connection.py:84
Integration: Home Assistant WebSocket API (documentation, issues)
First occurred: 13:33:15 (1 occurrences)
Last logged: 13:33:15

[1878560904] Received invalid command: browser_mod/connect
````
and
````
Logger: homeassistant.components.websocket_api.http.connection
Source: custom_components/browser_mod/helpers.py:47
Integration: Home Assistant WebSocket API (documentation, issues)
First occurred: 13:33:33 (13 occurrences)
Last logged: 13:34:20

[1854267080] Error handling message: Unknown error
[1854141728] Error handling message: Unknown error
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/websocket_api/connection.py", line 95, in async_handle
    handler(self.hass, self, schema(msg))
  File "/config/custom_components/browser_mod/connection.py", line 42, in handle_update
    devices[deviceID].update(msg.get("data", None))
  File "/config/custom_components/browser_mod/connection.py", line 83, in update
    self.sensor = self.sensor or create_entity(
  File "/config/custom_components/browser_mod/helpers.py", line 47, in create_entity
    adder = hass.data[DOMAIN][DATA_ADDERS][platform]
KeyError: 'sensor'
`````

maybe that is relevant.

In general, browser mod seems to word. Popups appearing. I do not use css.

**Comment by KTibow** [comment]
Those errors are normal and usually browser_mod still works.

**Comment by Pirol62** [comment]
Ok, the errors during restart - d'accord
But the "window.queueMicrotask is not a function" seem to be a wrong function call and might be possible to fix
It appears naturally very often when browsing through the dashboards.

**Comment by KTibow** [comment]
Dupe of #157

**Comment by Pirol62** [comment]
The solution did not work here.
I added the suggested line and still have the error.

**Comment by KTibow** [comment]
The `KeyError: 'sensor'` error is normal, everyone gets it.

**Comment by Pirol62** [comment]
Hi, just a friendly reminder. The workaround did not work for me and the TypeError: window.queueMicrotask is still valid in 2020.12.2.
Anyway: Have a happy new year and thanks for your huge engagement!!

**Comment by KTibow** [comment]
> and the TypeError: window.queueMicrotask is still valid

Can you share a bit of your log?

**Comment by petarvucetin** [comment]
This is mine:

`2021-01-01 10:44:02 ERROR (MainThread) [frontend.js.latest.202012120] https://[nabucasa].ui.nabu.casa/browser_mod.js:149:7364 Uncaught TypeError: window.queueMicrotask is not a function
2021-01-01 10:44:12 ERROR (MainThread) [frontend.js.latest.202012120] https://[nabucasa].ui.nabu.casa/browser_mod.js:149:7364 Uncaught TypeError: window.queueMicrotask is not a function
2021-01-01 10:44:16 ERROR (MainThread) [frontend.js.latest.202012120] https://[nabucasa].ui.nabu.casa/browser_mod.js:149:7364 Uncaught TypeError: window.queueMicrotask is not a function
2021-01-01 10:44:21 ERROR (MainThread) [frontend.js.latest.202012120] https://[nabucasa].ui.nabu.casa/browser_mod.js:149:7364 Uncaught TypeError: window.queueMicrotask is not a function
2021-01-01 10:44:22 ERROR (MainThread) [frontend.js.latest.202012120] https://[nabucasa].ui.nabu.casa/browser_mod.js:149:7364 Uncaught TypeError: window.queueMicrotask is not a function
2021-01-01 10:44:32 ERROR (MainThread) [frontend.js.latest.202012120] https://[nabucasa].ui.nabu.casa/browser_mod.js:149:7364 Uncaught TypeError: window.queueMicrotask is not a function
2021-01-01 10:44:42 ERROR (MainThread) [frontend.js.latest.202012120] https://[nabucasa].ui.nabu.casa/browser_mod.js:149:7364 Uncaught TypeError: window.queueMicrotask is not a function`

**Comment by Jpsy** [comment]
I witnessed this problem on an Amazon Fire Tablet.
It is the result of an outdated Webview component.
Android Webview supports queueMicrotask since v0.71. My Fire was running v0.70.

I did two things to solve the problem:

I updated Webview through Google Play Store. (I have manually installed Play Store on my Fire Tablet).
I updated Fire OS from v7.3.1.4 to the latest v7.3.1.6.
I now have Webview v0.75 and the problem is gone.
